auth_providers/oauth2-google.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "oauth2-google",
+    "type": "oauth2-google",
+    "config": {
+        "clientId": "653015965076-bnbqib0g3555nfmghhtc2ta352meadg0.apps.googleusercontent.com"
```

Comment:
Are this or the clientSecret considered sensitive data? If so we probably don't want it to sit in our public repo.

Reply:
the clientId is public, the secret I had to pre-create in the UI and then refer to it by name in the values folder.